assets/styles/styles.css

```diff
@@ -0,0 +1,347 @@
+body {
+  background-color: #f4f4f7;
+
+  /* background-color: seagreen;  */
+  font-family: 'Roboto Condensed', sans-serif;
+}
+
+.sec {
+  /* width: 500px;
+   min-width: 200px; */
+  min-height: 20vh;
+}
+
+.sec1 {
+  background-color: seagreen;
+}
+
+.sec2 {
+  background-color: sienna;
+}
+
+.sec3 {
+  background-color: slateblue;
+}
+
+.sec4 {
+  background-color: rgb(122, 121, 156);
+}
+
+.sec5 {
+  background-color: rgba(235, 11, 255, 0.945);
+}
+
+.twot {
+  /* margin-top: 100px !important; */
+}
+
+img {
+  /* width: 100%;  */
+}
+
+:root {
+  --header-label-color: #9a9a9a;
+  --p-color: #333;
+  --opinion-author-text-color: #555;
+  --main-theme-color: #e11c1f;
+  --nav-bg-color: #ff1c1f;
+}
+
+.white-filter {
+  filter: invert(99%) sepia(61%) saturate(86%) hue-rotate(219deg) brightness(116%) contrast(100%);
+}
+
+.orange-filter {
+  filter: invert(30%) sepia(57%) saturate(7215%) hue-rotate(346deg) brightness(99%) contrast(112%);
+}
+
+/* base */
+a {
+  text-decoration: none;
+  color: black;
+}
+
+/*
+a:hover {
+    text-decoration: none;
+    color: black;
+    border-bottom: 1px solid red;
+} */
```

Comment:
- [ ] It’s better to delete **commented piece of codes**. If you need to review a previous version of your code, use git history ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/general/no_inline_comments.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/66243643/))  
:x: **commented piece of codes** /* `.nav { color: red;}` */  
:heavy_check_mark: **comments about code** /*Navigation */